src/Metaplex.ts

```diff
@@ -39,7 +39,7 @@ export class Metaplex {
   public readonly options: MetaplexOptions;
 
   /** Encapsulates the identity of the users interacting with the SDK. */
-  protected identityDriver: IdentityDriver;
```

Comment:
Oh no we don't want to do that haha. If you could just revert the changes of that file, I'll propose another way to fix the type errors.